hazelcast/src/main/java/com/hazelcast/splitbrainprotection/impl/SplitBrainProtectionImpl.java

```diff
@@ -162,7 +162,7 @@ public SplitBrainProtectionConfig getConfig() {
     }
 
     @Override
-    public boolean isMinimumClusterSizeSatisfied() {
+    public boolean hasMinimumSize() {
         return splitBrainProtectionState == SplitBrainProtectionState.MIN_CLUSTER_SIZE_SATISFIED;
```

Comment:
minor: This `SplitBrainProtectionState.MIN_CLUSTER_SIZE_SATISFIED` can also be renamed.